.github/workflows/push_ci_docker_base.yml

```
@@ -0,0 +1,23 @@
+name: Push CI Docker Base
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'ci-docker-base/**'
```

could you also have this run when the workflow itself changes?
```suggestion
      - '.github/workflows/push_ci_docker_base.yml'
      - 'ci-docker-base/**'
```